src/Rector/Closure/MinimalSharedStringSolver.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\Closure;
+
+/**
+ * Class Solver copied from https://github.com/Triun/PHP-Longest-Common-Substring
+ */
+final class MinimalSharedStringSolver
```

Comment:
@staabm This needs a test, as there is lot of tiny coding happening here.
Could you add it?

Reply:
sure, here you are: https://github.com/rectorphp/rector-symfony/pull/380